test/ubi/UBISchemeCycle.test.js

```diff
@@ -0,0 +1,196 @@
+const MarketMaker = artifacts.require("GoodMarketMaker");
```

Comment:
Should we adapt this tests to hardhat in this PR or later?

Reply:
there's an issue for that